sql/solutions/lesson3.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: Introduction to SQL Solutions
+---
+# Lesson 3 solutions to exercises
+
+## L3.1 How many people are not living in their country of birth.
+
+```SQL
+SELECT COUNT(*)
+  FROM people
+ WHERE country_of_birth IS NOT country_of_residence;
+```
+
+##  L3.2 What is the average age of the people not living in their country of birth.
+
+```SQL
+SELECT AVG(age)
+  FROM people
+ WHERE country_of_birth IS NOT country_of_residence;
+```
+
+## L3.3 Show the age of the oldest person. (Find a different way to the one use in Lesson 2).
+
+Solution 1:
+```SQL
+SELECT MAX(age)
+  FROM people;
+```
+
+Solution 2:
+```SQL
+SELECT age
+  FROM people
+ ORDER BY age DESC
+ LIMIT 1;
```

Comment:
Solution 2 is the same as in lesson 2, and the exercise is asking to find a different way :)
I propose to remove it or rename it to: "Lesson 2 solution:"